digdag-core/src/main/java/io/digdag/core/schedule/ScheduleExecutor.java

```diff
@@ -199,8 +199,13 @@ private void runSchedule(ScheduleControl lockedSched)
 
             Config scheduleConfig = SchedulerManager.getScheduleConfig(def);
             boolean skipOnOvertime = scheduleConfig.get("skip_on_overtime", boolean.class, false);
+            boolean disableBackfill = scheduleConfig.get("disable_backfill", boolean.class, false);
 
-            if (!activeAttempts.isEmpty() && skipOnOvertime) {
+            if (disableBackfill && now.isAfter(sched.getNextRunTime())) {
+                logger.info("Now={} is later than scheduled time={}. Skipping this schedule: {}", now, sched.getNextScheduleTime(), sched);
```

Comment:
Could you modify this message to say that the schedule will be skipped since backfill is disable?

Reply:
done.